bin/sdk-cli.js

```diff
@@ -28,54 +75,47 @@ yargs
 	.command( {
 		command: 'gutenberg',
 		desc: 'Build a Gutenberg extension',
-		builder: yargs => yargs.options( {
-			'mode': {
-				alias: 'm',
-				description: 'Choose the way how assets are optimized.',
-				type: 'string',
-				choices: [ 'production', 'development' ],
-				default: 'production',
-				requiresArg: true,
-			},
-			'editor-script': {
-				description: 'Entry for editor-side JavaScript file',
-				type: 'string',
-				required: true,
-				coerce: value => path.resolve( __dirname, '../', value ),
-				requiresArg: true,
-			},
-			'view-script': {
-				description: 'Entry for rendered-page-side JavaScript file',
-				type: 'string',
-				coerce: value => path.resolve( __dirname, '../', value ),
-				requiresArg: true,
-			},
-			'output-dir': {
-				alias: 'o',
-				description: 'Output directory for the built assets. Intermediate directories are created as required.',
-				type: 'string',
-				coerce: path.resolve,
-				requiresArg: true,
-			},
-			'output-editor-file': {
-				description: 'Name of the built editor script output file (without the file extension).',
-				type: 'string',
-				requiresArg: true,
-			},
-			'output-view-file': {
-				description: 'Name of the built view script output file (without the file extension).',
-				type: 'string',
-				requiresArg: true,
-			},
-			'watch': {
-				alias: 'w',
-				description: 'Whether to watch for changes and automatically rebuild.',
-				type: 'boolean',
-			},
-		} ),
-		handler: argv => gutenberg.compile( argv )
+		builder: yargs =>
+			yargs.options( {
+				'editor-script': {
+					description: 'Entry for editor-side JavaScript file',
+					type: 'string',
+					required: true,
+					coerce: value => path.resolve( __dirname, '../', value ),
```

Comment:
Not related with the PR, but occurred to me now: the `*-script` args use the same `coerce` callback - should we define it separately and use it for both?

Reply:
🤷‍♂️ is the point to not write it twice? I don't understand `coerce` so I don't have a real opinion on the matter.

Reply:
It's not about `coerce` - it's just about not defining the same callback twice when we can define it just once and just reuse it.